apps/cowswap-frontend/public/emergency.js

```diff
@@ -7,6 +7,44 @@ if (window.location.host === 'barn.cowswap.exchange') {
   window.location.href = 'https://barn.cow.fi'
 }
 
+/**
+ * Remove old versions of the local storage atom stores
+ * We rely on the fact that store names are in the format {name}Atom:v{version}
+ * Since outdated versions of the stores are not used anymore, we should remove them to not exceed the storage limit
+ */
+;(function () {
+  const atomStoreRegex = /^(.+Atom):v(\d{1,3})$/
```

Comment:
What about atom names that don't match the `Atom:v` pattern?

(VSCode doesn't seem to match negative lookaheads, but anyway)

![image](https://github.com/cowprotocol/cowswap/assets/43217/cdc74866-0bd8-4dbe-9096-3c3bdd3348bc)

Reply:
Great point!
I would suggest to use two patterns:
 - `*-atom:v0`
 - `*Atom:v0`
 
 Wdyt?

Reply:
We have more that don't follow that pattern. Examples:
- warningBannerVisibility:v0
- appDataUploadQueue:v1
- staticPermitCache:v3
- userPermitCache:v1
- permittableTokens:v3
- safeTokenWarning:v0
- eth-flow-in-flight-order-ids:v0
- limitOrdersZeroBalanceBanner:v0
- fallbackHandlerVerification:v2
- twap-orders-list:v1
- twap-part-orders-list:v1
- tokensLoadedFromChain:v0
- closableBanners:v0
- hw-account-index:v1